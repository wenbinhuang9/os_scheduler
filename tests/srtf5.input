3 4 10
1 4 9
2 4 8
5 1 15