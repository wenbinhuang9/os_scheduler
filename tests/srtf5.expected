1 21 8 17
2 12 0 8
3 31 17 27
5 43 27 42
13 24